src/api/permissions.js

```diff
@@ -20,7 +20,6 @@ export const getMyGroupPermissions = () => (
     method: METHODS.GET,
     mockResponse: MY_PERMISSIONS_PAYLOAD_OK,
     useAuthentication: true,
```

Comment:
why you need to delete useCredentials? It's releated with this task? are you sure that this don't take unexepected issue?

Reply:
is see there is a PR of @zonia3000 about that https://github.com/entando/app-builder/pull/1300

Reply:
no any other API call uses `useCredentials: true`. Removing it does not interfere with anything, its simply not necessary. On the other hand, when using local app-builder with remote de-app instance, that line causes CORS errors, so it has to be definitely removed.

Reply:
ok, talk with @zonia3000 and agree on her PR. She has made the same fix.

Reply:
Hi, it is ok for me to remove `useCredentials` on this PR. I can close (without merging) my PR when this one will be merged.